R/html_dependencies.R

```diff
@@ -38,24 +38,62 @@ html_dependency_jqueryui <- function() {
 
 # Create an HTML dependency for Bootstrap
 #' @rdname html-dependencies
+#' @inheritParams html_document
+#' @param version Bootstrap version. See `bootstrap_version` in [html_document]
 #' @export
-html_dependency_bootstrap <- function(theme) {
+html_dependency_bootstrap <- function(theme, version = c("3", "4", "4-3")) {
+
+  version <- bootstrap_version_normalize(version)
+  theme <- bootswatch_theme_normalize(theme, version)
+
+  if (version == 3) {
+    return(
+      list(htmlDependency(
```

Comment:
Let's not return a `list` here, as doing so will break backward compatibility. However, `4`/`4-3` can return a list. Also consider modifying htmltools to accept nested lists of html dependencies.